app/assets/javascripts/short-form/components/preferenceComponent.js.coffee

```diff
@@ -52,10 +54,13 @@ angular.module('dahlia.components')
 
       # For 588 Mission preference, we need to override some defaults.
       @isEmploymentDisability = =>
-        @title == 'Employment/Disability Preference'
+        @title == 'Employment or Disability Preference'
```

Comment:
Added flagforI18n below so that `grunt translations` wouldn't delete the description.

I had to make this update here too to check that change

Reply:
Should this be changed back before merging?